### PR TITLE
Update roosevelt and prettify json

### LINKED ISSUE #172: Dependencies printed on one line
```
  "engineStrict": true,
  "dependencies": {"roosevelt":"~0.12.0","teddy":"~0.4.27","roosevelt-less":"~0.7.0","roosevelt-uglify":"~0.3.0"},
  "devDependencies": {
    "husky": "~1.1.4",
    "lint-staged": "~8.0.4",
    "nodemon": "~1.18.6",
    "standard": "~12.0.1",
    "stylelint": "~9.8.0",
    "stylelint-config-standard": "~18.2.0"
  },
```

Workaround: easily solved by running `npm i` after you generate your app, but still annoying if you want to edit the dependencies by hand first.

### LINKED ISSUE #208: HTTPS params are outdated
Sample apps with https enabled get generated with old Roosevelt v0.11.x HTTPS params.

They need to be updated to [their current implementation](https://github.com/rooseveltframework/roosevelt#https-parameters)

### PR DESCRIPTION
**Pull request by lyomr**
This PR: 
- Updates the generator to latest version of Roosevelt (v.14)
- Fixes outdated http params (closes #208) 
- Fixes one liner dependency print (close #172)
- Updates logger params to latest
- Updates CHANGELOG
- Add dev-install and dev-prune scripts

### CODE REVIEW

File: generators/app/index.js [hunk 1]
```diff
@@ -590,15 +641,7 @@ module.exports = class extends Generator {
...
-        enableHTTPS: this.enableHTTPS,
-        httpsOnly: this.httpsOnly,
-        httpsPort: this.httpsPort,
-        pfx: this.pfx,
-        keyPath: this.keyPath,
-        passphrase: this.passphrase,
-        ca: this.ca,
-        requestCert: this.requestCert,
-        rejectUnauthorized: this.rejectUnauthorized,
+        https: this.httpsParams,
```
Comment (kethinov) [inline generators/app/index.js]: when creating a fresh app, there are still issues:

```
⚠️   Missing script "dev-install"!
⚠️   Missing script "dev-prune"!
```

config still needs the latest scripts https://github.com/rooseveltframework/roosevelt/blob/master/lib/defaults/scripts.json

(not relevant to this line, just dropping the comment here to have somewhere to put it)

Comment (lyomr) [reply generators/app/index.js]: Fixed with 9ea6f32

File: generators/app/templates/_package.json [hunk 2]
```diff
@@ -27,10 +27,13 @@
     "routers": false,
     "cleanTimer": 604800000,
     "logging": {
```
Comment (Autre31415) [inline generators/app/templates/_package.json]: The `frontendReload` params need to be added https://github.com/rooseveltframework/roosevelt/blob/master/lib/defaults/config.json#L50

Comment (lyomr) [reply generators/app/templates/_package.json]: All set with 93046b3

**Pull request closed without merging**